scripts/issue/78/grant_access_ibbtc_peak.py

```diff
@@ -4,16 +4,17 @@
 
 TREASURY_OPS = registry.eth.badger_wallets.treasury_ops_multisig
 
-def main():
+def main(peak="badger"):
     safe = GreatApeSafe(registry.ibbtc.dfdBadgerShared)
     safe.init_badger()
```

Comment:
minor comment, in this case it is not necessary, just slows down the script

Reply:
Ah, I had added it because I thought for a second I could use the whitelisting functions from it. Turns out I couldn't and forgot to remove it. Thanks for the heads up!